third_party/multiarch/qemu-user-static/register/register.sh

```diff
@@ -19,13 +19,13 @@ entries="aarch64 aarch64_be alpha arm armeb hppa m68k microblaze microblazeel mi
 if [ "${1}" = "--reset" ]; then
     shift
     (
-    cd /proc/sys/fs/binfmt_misc
+    cd /proc/sys/fs/binfmt_misc || { echo "Could not cd to /proc/sys/fs/binfmt_misc"; exit 1; }
```

Comment:
For reference - [SC2164](https://github.com/koalaman/shellcheck/wiki/SC2164) - _"Use cd ... || exit in case cd fails."_

Reply:
Do we need to move to `third_party/forked` if we modify this? @spiffxp